### ISSUE TITLE
[Gundrak] Drakkari Elemental Missing Mojo Puddle skill

### ISSUE THREAD

**Issue by AlexCSCP** [dalaranwow/dalaran-wow#5505]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The boss only use mojo volley.
**Expected behaviour**: Tell us what should happen instead
The boss should use a skill that create mojo puddles, which deal big damage to those standing on it.
**Steps to reproduce the problem**:

1. Go to gundrak
2. Fight the boss
3. See that the elemental only throw mojo.

**Include proofs for this behaviour**
http://www.wowhead.com/npc=29573/drakkari-elemental
"he has a fountain spray that shoots blobs of mojo around the room at other group members. these leave puddles that can wreck this fight."
http://wowwiki.wikia.com/wiki/Drakkari_Colossus?oldid=2309602
"but every .5 of a second he will cast "pool of Mojo" on three people in the party"

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by AlexCSCP** [comment]
I don't remember seeing it doing the random charge skill, but it may just no be clear enough to notice.
